tools/devtool

```diff
@@ -72,7 +72,7 @@
 DEVCTR_IMAGE_NO_TAG="public.ecr.aws/firecracker/fcuvm"
 
 # Development container tag
-DEVCTR_IMAGE_TAG=${DEVCTR_IMAGE_TAG:-v67}
```

Comment:
Is this for testing the changes?

I'm not sure we want this. This would mean older supported versions will run with the latest version, which may result in errors.

Also the moment a new version is created, we will start using it (even if that is not intended, if we create for example a fix for an old version `v67_2`

Reply:
that's just for making the CI run, hence the Draft PR. I will fix that once I create the container image and push it on ECR